## 52% of end-end tests have passed
```
Total executed tests: 56
Failed tests: 27
Expected to fail tests: 0
Passed tests: 29
```
```
IDs of failed tests: 702845,702840,702894,703086,702947,702958,702786,702731,702844,703202,741054,702841,702807,702839,702808,702809,702838,704615,702948,702859,703629,702957,702843,702732,703297,703495,703194 
```
<h3>Failed tests (27)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/check-bridge-to-chain-id&PR_ID=21980&APK_NAME=21980.apk&TR_CASE_IDS=702845,702840,702894,703086,702947,702958,702786,702731,702844,703202,741054,702841,702807,702839,702808,702809,702838,704615,702948,702859,703629,702957,702843,702732,703297,703495,703194">Rerun failed tests</a></li><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772353">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div><div>Device 2: Element Button text is equal to 2 </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:278: in test_1_1_chat_pin_messages
    self.errors.verify_no_errors()
base_test_case.py:179: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: Message 4 is not pinned in chat after unpinning previous one
E    Device 2: Message 4 is not pinned in chat after unpinning previous one
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZjOUnP0lhVmfVakX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpAV2tQXeMWeXdqlx">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772385">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:383: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:84: in create_user
    self.set_password(password)
../views/sign_in_view.py:69: in set_password
    input_elements[1].click()
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click
    self._execute(Command.CLICK_ELEMENT)
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element/00000000-0000-000c-ffff-ffff000000a5/click' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element/00000000-0000-000c-ffff-ffff000000a5/click' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/@appium/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at runMicrotasks (<anonymous>)
E       at processTicksAndRejections (node:internal/process/task_queues:96:5)
E       at AndroidUiautomator2Driver.click (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/lib/commands/element.js:136:5)
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772384">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>## New multiaccount is created successfully!</div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:383: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:84: in create_user
    self.set_password(password)
../views/sign_in_view.py:69: in set_password
    input_elements[1].click()
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:94: in click
    self._execute(Command.CLICK_ELEMENT)
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element/00000000-0000-000c-ffff-ffff000000a5/click' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element/00000000-0000-000c-ffff-ffff000000a5/click' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/node_modules/@appium/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at runMicrotasks (<anonymous>)
E       at processTicksAndRejections (node:internal/process/task_queues:96:5)
E       at AndroidUiautomator2Driver.click (/root/lambda/shared-tools/volume/uiautomator/uiautomator-2.32.3/lib/commands/element.js:136:5)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES0xYEpuyzyntH38fb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES3EVmL8LbUItMrRTm">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772407">test_fallback_add_key_pair</a>, id: 741054 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping left on element Button </div><div>Device 1: Find `Button` by `xpath`: `//android.view.ViewGroup[contains(@content-desc,'Imported account')]` </div></blockquote></p>

```
critical/test_fallback.py:212: in test_fallback_add_key_pair
    wallet_1.get_account_element(account_name=imported_key_pair_account_name).swipe_left_on_element()
../views/base_element.py:281: in swipe_left_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:274: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//android.view.ViewGroup[contains(@content-desc,'Imported account')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772389">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shrj1W8oA2jP6ufVyjjv')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772364">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shrj1W8oA2jP6ufVyjjv')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES4z8P5h7LnFOWomCK">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLagYgCNFELR7aQfy">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESx0ySJYHXxiaVtwV1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772365">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shrj1W8oA2jP6ufVyjjv')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772354">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shrj1W8oA2jP6ufVyjjv')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772390">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shrj1W8oA2jP6ufVyjjv')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772393">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:63: in prepare_devices
    self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shrj1W8oA2jP6ufVyjjv')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772370">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772381">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772386">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772373">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Youtube links preview is not loaded on LambdaTest emulators, needs investigation]]</b><br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772371">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772369">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772368">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESxCGTrURlH23NyUrb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESNK0iGkMhBVMIRFEO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772398">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772379">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772372">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772388">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:325: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shZ4di5qtzgTJKuVi5hi')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772374">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'sheBVuXKVqbKkDvZS6tQ')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772363">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'sheBVuXKVqbKkDvZS6tQ')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES7s9JK7omY2XdZSNy">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESizMuXbqsAeJgd5HC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772366">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'sheBVuXKVqbKkDvZS6tQ')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772383">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'sheBVuXKVqbKkDvZS6tQ')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772396">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:867: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'sheBVuXKVqbKkDvZS6tQ')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772382">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:251: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:388: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:167: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:164: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:154: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:138: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'shnH6RsSu2gncvgkn5nZ')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLaR7c2Vgng1IxWxG">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLY78D2IsZM3vF4Em">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (29)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772377">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESp4GPmw0SOSduKkWm">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESbQ3Uobr8Ey9KViP6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772376">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESp4GPmw0SOSduKkWm">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESbQ3Uobr8Ey9KViP6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772359">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESp4GPmw0SOSduKkWm">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESbQ3Uobr8Ey9KViP6">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772378">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZjOUnP0lhVmfVakX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpAV2tQXeMWeXdqlx">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772352">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZjOUnP0lhVmfVakX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpAV2tQXeMWeXdqlx">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772357">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZjOUnP0lhVmfVakX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpAV2tQXeMWeXdqlx">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772355">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZjOUnP0lhVmfVakX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpAV2tQXeMWeXdqlx">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772367">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZjOUnP0lhVmfVakX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpAV2tQXeMWeXdqlx">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772360">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZjOUnP0lhVmfVakX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpAV2tQXeMWeXdqlx">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772392">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZjOUnP0lhVmfVakX">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpAV2tQXeMWeXdqlx">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772397">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESeoDbgzn4me2y6aZ1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772358">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESeoDbgzn4me2y6aZ1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772402">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESeoDbgzn4me2y6aZ1">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772401">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772406">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772394">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES1V42I4lIqfjBDtvg">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES01XPcAqpLg15V3i0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772361">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES1V42I4lIqfjBDtvg">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES01XPcAqpLg15V3i0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772362">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES1V42I4lIqfjBDtvg">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES01XPcAqpLg15V3i0">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772387">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ3XjvbgYjsN5iG91">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772356">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ3XjvbgYjsN5iG91">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772375">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ3XjvbgYjsN5iG91">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772380">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ3XjvbgYjsN5iG91">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772391">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ3XjvbgYjsN5iG91">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772395">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ3XjvbgYjsN5iG91">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772403">test_fallback_sync_with_error</a>, id: 740220 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772404">test_fallback_with_correct_seed_phrase</a>, id: 740221 </b></td></tr><tr><td></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772405">test_fallback_validate_seed_phrase</a>, id: 740222 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772400">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772399">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table></details>

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v9.3.0",
-    "commit-sha1": "bec0c3733eb9629153bc3df3d6294cec6a23af09",
-    "src-sha256": "0vj9vhmlf8csnpy8b0xiny425yzpl4wj1z873igb5av5hqllabf3"
+    "version": "v10.0.0",
+    "commit-sha1": "dcce8e9b8aec04876b854eacb56d9de95fd96a59",
+    "src-sha256": "008z0wrf6slf32iy3navq4zznfy6lam72lpqhxqwl8705hk4a4mj"
 }

